### ISSUE TITLE
Extension doesn't work with latest MediaWiki

### ISSUE THREAD

**Issue by rail01** [lkucharczyk/mediawiki-PortableInfobox#28]
Attempt to create page with `<infobox>` tag in it causes Internal Error with code `[XsJD3VEHhKj428zzi@KyfAAAAAg] 2020-05-18 08:14:22: Fatal exception of type "Error"`.

Tested on `MediaWiki 1.35.0-alpha (89ab2a1)`

**Comment by rail01** [comment]
Additional error data:

```
[XsJHoQpn3AzhozY2yDh2JwAAAAo] /index.php?title=Szablon:Test&action=submit Error from line 14 of /opt/lampp/htdocs/extensions/PortableInfobox/includes/services/PortableInfoboxRenderService.php: Class 'LightnCandy' not found

Backtrace:

#0 /opt/lampp/htdocs/extensions/PortableInfobox/includes/controllers/PortableInfoboxParserTagController.php(71): PortableInfoboxRenderService->__construct()
#1 /opt/lampp/htdocs/extensions/PortableInfobox/includes/controllers/PortableInfoboxParserTagController.php(127): PortableInfoboxParserTagController->render(string, Parser, PPFrame_Hash, array)
#2 /opt/lampp/htdocs/includes/parser/Parser.php(3858): PortableInfoboxParserTagController->renderInfobox(string, array, Parser, PPFrame_Hash)
#3 /opt/lampp/htdocs/includes/parser/PPFrame_Hash.php(327): Parser->extensionSubstitution(array, PPFrame_Hash)
#4 /opt/lampp/htdocs/includes/parser/Parser.php(2862): PPFrame_Hash->expand(PPNode_Hash_Tree, integer)
#5 /opt/lampp/htdocs/includes/parser/Parser.php(1548): Parser->replaceVariables(string)
#6 /opt/lampp/htdocs/includes/parser/Parser.php(635): Parser->internalParse(string)
#7 /opt/lampp/htdocs/includes/content/WikitextContent.php(368): Parser->parse(string, Title, ParserOptions, boolean, boolean, NULL)
#8 /opt/lampp/htdocs/includes/content/AbstractContent.php(565): WikitextContent->fillParserOutput(Title, NULL, ParserOptions, boolean, ParserOutput)
#9 /opt/lampp/htdocs/includes/Revision/RenderedRevision.php(264): AbstractContent->getParserOutput(Title, NULL, ParserOptions, boolean)
#10 /opt/lampp/htdocs/includes/Revision/RenderedRevision.php(236): MediaWiki\Revision\RenderedRevision->getSlotParserOutputUncached(WikitextContent, boolean)
#11 /opt/lampp/htdocs/includes/Revision/RevisionRenderer.php(215): MediaWiki\Revision\RenderedRevision->getSlotParserOutput(string)
#12 /opt/lampp/htdocs/includes/Revision/RevisionRenderer.php(152): MediaWiki\Revision\RevisionRenderer->combineSlotOutput(MediaWiki\Revision\RenderedRevision, array)
#13 [internal function]: MediaWiki\Revision\RevisionRenderer->MediaWiki\Revision\{closure}(MediaWiki\Revision\RenderedRevision, array)
#14 /opt/lampp/htdocs/includes/Revision/RenderedRevision.php(198): call_user_func(Closure, MediaWiki\Revision\RenderedRevision, array)
#15 /opt/lampp/htdocs/includes/Storage/DerivedPageDataUpdater.php(1301): MediaWiki\Revision\RenderedRevision->getRevisionParserOutput()
#16 [internal function]: MediaWiki\Storage\DerivedPageDataUpdater->getCanonicalParserOutput()
#17 /opt/lampp/htdocs/includes/edit/PreparedEdit.php(104): call_user_func(array)
#18 /opt/lampp/htdocs/includes/edit/PreparedEdit.php(119): MediaWiki\Edit\PreparedEdit->getOutput()
#19 /opt/lampp/htdocs/includes/Storage/DerivedPageDataUpdater.php(1279): MediaWiki\Edit\PreparedEdit->__get(string)
#20 /opt/lampp/htdocs/includes/page/WikiPage.php(2064): MediaWiki\Storage\DerivedPageDataUpdater->getPreparedEdit()
#21 /opt/lampp/htdocs/extensions/ConfirmEdit/SimpleCaptcha/SimpleCaptcha.php(591): WikiPage->prepareContentForEdit(WikitextContent)
#22 /opt/lampp/htdocs/extensions/ConfirmEdit/SimpleCaptcha/SimpleCaptcha.php(810): SimpleCaptcha->shouldCheck(WikiPage, WikitextContent, string, DerivativeContext)
#23 /opt/lampp/htdocs/extensions/ConfirmEdit/SimpleCaptcha/SimpleCaptcha.php(846): SimpleCaptcha->doConfirmEdit(WikiPage, WikitextContent, string, DerivativeContext, User)
#24 /opt/lampp/htdocs/extensions/ConfirmEdit/includes/ConfirmEditHooks.php(38): SimpleCaptcha->confirmEditMerged(DerivativeContext, WikitextContent, Status, string, User, boolean)
#25 /opt/lampp/htdocs/includes/HookContainer/HookContainer.php(313): ConfirmEditHooks::confirmEditMerged(DerivativeContext, WikitextContent, Status, string, User, boolean)
#26 /opt/lampp/htdocs/includes/HookContainer/HookContainer.php(125): MediaWiki\HookContainer\HookContainer->callLegacyHook(string, array, array, array)
#27 /opt/lampp/htdocs/includes/Hooks.php(136): MediaWiki\HookContainer\HookContainer->run(string, array, array)
#28 /opt/lampp/htdocs/includes/EditPage.php(1931): Hooks::run(string, array)
#29 /opt/lampp/htdocs/includes/EditPage.php(2260): EditPage->runPostMergeFilters(WikitextContent, Status, User)
#30 /opt/lampp/htdocs/includes/EditPage.php(1748): EditPage->internalAttemptSave(NULL, boolean)
#31 /opt/lampp/htdocs/includes/EditPage.php(728): EditPage->attemptSave(NULL)
#32 /opt/lampp/htdocs/includes/actions/EditAction.php(59): EditPage->edit()
#33 /opt/lampp/htdocs/includes/actions/SubmitAction.php(38): EditAction->show()
#34 /opt/lampp/htdocs/includes/MediaWiki.php(519): SubmitAction->show()
#35 /opt/lampp/htdocs/includes/MediaWiki.php(305): MediaWiki->performAction(Article, Title)
#36 /opt/lampp/htdocs/includes/MediaWiki.php(973): MediaWiki->performRequest()
#37 /opt/lampp/htdocs/includes/MediaWiki.php(535): MediaWiki->main()
#38 /opt/lampp/htdocs/index.php(47): MediaWiki->run()
#39 {main}
```

**Comment by rail01** [comment]
Fixed in #29
